pkg/cli/image/manifest/manifest.go

```diff
@@ -87,6 +87,7 @@ func (o *SecurityOptions) Context() (*registryclient.Context, error) {
 	}
 	context, err := o.NewContext()
 	o.CachedContext = context
+	o.CachedContext.Retries = 3
```

Comment:
Maybe it's better to do it as the last step in `o.NewContext()`?

Reply:
`Context()` is "I want the defaults for this command" while `NewContext` is "read from options only".  So Context